festim/generic_simulation.py

```diff
@@ -270,6 +270,26 @@ def initialise(self):
 
         self.h_transport_problem.initialise(self.mesh, self.materials, self.dt)
 
+        # raise warning if SurfaceFlux is used with non-cartesian meshes
+        for export in self.exports.exports:
```

Comment:
just to remind, `.exports` will be deprecated

Reply:
Yes I probably have some merge conflicts I need to take care of